website/js/base.js

```diff
@@ -91,20 +82,6 @@
                 });
             }
         });
-
-        var n = d.getElementsByTagName("script")[0],
-            s = d.createElement("script"),
-            f = function () { n.parentNode.insertBefore(s, n); };
-        s.type = "text/javascript";
-        s.async = true;
-        s.src = "/js/metrika.js";
-        if (window.location.hostname.endsWith('clickhouse.com')) {
-            if (w.opera == "[object Opera]") {
-                d.addEventListener("DOMContentLoaded", f, false);
-            } else {
-                f();
-            }
-        }
     })(document, window, "yandex_metrika_callbacks2");
```

Comment:
Can remove this too: "yandex_metrika_callbacks2"

But note that the function signature still expects it (line 70) and a line below as well

Reply:
as far as I understand this is just an obscure way to lazy load they provided with their snippet
this definitely deserves a refactoring, but it could probably be done separately some day...